components/layout/src/zoom.ts

```diff
@@ -60,7 +60,8 @@ export class HPCCZoomElement extends HPCCSVGElement {
             this.x = transform.x;
             this.y = transform.y;
             this.scale = transform.k;
-            this.$emit("changed");
+            this.$emit("change");
+            this.$emit("Change");
```

Comment:
Is this debounced?

Reply:
No - but there should only be 1 emit (I was testing against React and trying to get it to work with synthetic events).

Reply:
Resolved